instrumentation/opentelemetry-instrumentation-httpx/src/opentelemetry/instrumentation/httpx/__init__.py

```diff
@@ -731,8 +732,16 @@ def _instrument(self, **kwargs):
         self._original_async_client = httpx.AsyncClient
         request_hook = kwargs.get("request_hook")
         response_hook = kwargs.get("response_hook")
-        async_request_hook = kwargs.get("async_request_hook", request_hook)
-        async_response_hook = kwargs.get("async_response_hook", response_hook)
+        if iscoroutinefunction(request_hook):
+            async_request_hook = kwargs.get("async_request_hook", request_hook)
```

Comment:
From the previous review, I'm not sure we want this. I think the idea is to remove the default from the previous code here